inc/iso22133object.hpp

```diff
@@ -51,9 +50,11 @@ class TestObject {
 
 public:
 	TestObject(const std::string& listenIP = "0.0.0.0");
+	TestObject(int tcpSocket);
```

Comment:
The "tcpSocket" is what is called FD of file descriptor in the underlying C code? i was a bit confused by the name until I realised it was an int

Reply:
changed the name